### ISSUE TITLE
Do not create plugin's databases on "public" schema

### ISSUE THREAD

**Issue by matthieumarrast** [metwork-framework/mfbase#252]
Currently a database created for a mfbase plugin is related to the **public** schema.

Maybe it should be better if a plugin has its own schema.

For example when processing a `pg_dump -c` command for dumping database, we must provide the schema for which the database objects are related, so "public". As a consequence, it will write in dump file commands like `DROP SCHEMA public;`, which can be very dangerous !

Possible workaround solution:
- before installing plugin, create a file `create_schema.sql` in the "sql" folder, with content:
```
CREATE SCHEMA plugin_<plugin> AUTHORIZATION  plugin_<plugin>;
```
where \<plugin\> is the plugin name.

It will associate the plugin database to its own schema, then all the objetcs will be associated to this schema.

**Comment by matthieumarrast** [comment]
I raised the level to `critical` as it appears we cannot create a table to a new bootstraped plugin.

```
ERROR: permission denied for schema public
```

**Comment by matthieumarrast** [comment]
on metwork v2.0, we can create a TABLE after bootstraping a plugin. I will check on metwork 2.1.

**Comment by matthieumarrast** [comment]
PostgreSQL removed creation rights on the public schema from **v15**: `Remove PUBLIC creation permission on the public schema` (cf.  https://www.postgresql.org/docs/release/15.0/) for security reason.

So only mfbase from MetWork **v2.2** is involved.
An emergency fix is mandatory.

**Comment by matthieumarrast** [comment]
I found a solution for both issues, in `_create_plugin_database.sh`:

1. in **createdb** command, we must add the `--owner="plugin_${NAME}"` option, also the plugin's user becomes the owner of the database. The superuser "metwork" is no longer the owner of the database.
So the command `GRANT ALL PRIVILEGES ON DATABASE "plugin_${NAME}" to "plugin_${NAME}";` can be deleted as the user is now the owner of the database, it has all the privileges on it.

2. add a `CREATE SCHEMA "plugin_${NAME}"` command with the plugin's user on the plugin's database, also this schema will be used instead of "public". Indeed if we look for the `SHOW search_path` it returns `"$user", public...` meaning a schema name equals to the user name will be used prior.

**Comment by thebaptiste** [comment]
Closed by #264
